src/components/popup/details/purposes/purposes.jsx

```diff
@@ -161,10 +161,10 @@ export default class Purposes extends Component {
 						<div class={style.body}>
 							<LocalLabel localizeKey={`${currentPurposeLocalizePrefix}.description`} />
 							{!purposeIsTechnical &&
-							<a class={style.vendorLink} onClick={this.createOnShowVendors({ isCustom: false })}><LocalLabel localizeKey='showVendors'>Show full vendor list</LocalLabel></a>
+							<div><a class={style.vendorLink} onClick={this.createOnShowVendors({ isCustom: true })}><LocalLabel localizeKey='showCustomVendors'>Show full custom vendor list</LocalLabel></a></div>
 							}
 							{!purposeIsTechnical &&
```

Comment:
Są dwa razy te same ify, można to po prostu ograć fragmentem czyli
```
{!purposeIsTechnical &&
<>
<div><a class={style.vendorLink} onClick={this.createOnShowVendors({ isCustom: true })}><LocalLabel localizeKey='showCustomVendors'>Show full custom vendor list</LocalLabel></a></div>
<div><a class={style.vendorLink} onClick={this.createOnShowVendors({ isCustom: false })}><LocalLabel localizeKey='showVendors'>Show full vendor list</LocalLabel></a></div>
</>
}
```

Reply:
Unfortunately it doesn't work because we use Preact not React.

Reply:
I've just implemented it in different way

Reply:
Based on their [documentation](https://preactjs.com/guide/v10/components/#fragments) fragments are available in the same way as in react. But the array is good too :)